docs/src/modules.md

```diff
@@ -34,44 +34,47 @@ Other files needed by the module can also be stored within the
 
 ## Module template
 
-All modules should have an enable option created using `mkEnableTarget`.
-This is similar to
-[`mkEnableOption`](https://nix-community.github.io/docnix/reference/lib/options/lib-options-mkenableoption/)
-from the standard library, however it integrates with
-[`stylix.enable`](./options/nixos.md#stylixenable) and
-[`stylix.autoEnable`](./options/nixos.md#stylixautoenable)
-and generates more specific documentation.
+All modules should be created using `mkTarget`.
 
 A general format for modules is shown below.
 
 ```nix
+mkTarget:
 { config, lib, ... }:
+mkTarget
 {
-  options.stylix.targets.«name».enable =
-    config.lib.stylix.mkEnableTarget "«human readable name»" true;
+  name = "«target attribute name»";
+  humanName = "«human readable name»";
 
-  config =
-    lib.mkIf (config.stylix.enable && config.stylix.targets.«name».enable)
+  configElements = [
+    { colors }:
       {
         programs.«name».backgroundColor = config.lib.stylix.colors.base00;
       };
+  ];
```

Comment:
This has a syntax error, list element should be wrapped with `(`parens`)` because it contains whitespace.

Alternatively, you could use the new non-list style as it is a single element:

```suggestion
  configElements =
    { colors }:
    {
      programs.«name».backgroundColor = config.lib.stylix.colors.base00;
    };
```